chirp/chirp_common.py

```diff
@@ -87,7 +87,7 @@ def VALIDTONE(v):
 
 MODES = ("WFM", "FM", "NFM", "AM", "NAM", "DV", "USB", "LSB", "CW", "RTTY",
          "DIG", "PKT", "NCW", "NCWR", "CWR", "P25", "Auto", "RTTYR",
-         "FSK", "FSKR", "DMR", "DN")
+         "FSK", "FSKR", "DMR", "DN", "USER-L", "USER-U", "RTTY-L")
```

Comment:
These yaesu-specific "modes" need to be handled some other way. The existing driver already has problems with this (as you've probably noticed) so that needs to get fixed. I dunno what to map `USER` to, but `RTTY-L` likely needs to be just `RTTY` and with an extra knob that lets you select the `-L` variant as a separate flag or something. Or just collapse it all into `RTTY` and call it good :)

Reply:
The two radio types appear to use memory differently, and they really are unique. Use of just "RTTY won't work because the different USER- types require setting different areas of memory. I already fixed my driver to handle the added types that need to be included in the potential modes in chip_common. Chirp common also burped on pep8 which is why I made the other changes. I am absolutely stuck at this point in the bowels of git and github.